.github/workflows/test.yaml

```diff
@@ -58,6 +58,12 @@ jobs:
         run: |
           go install ./cmd/swagger
 
+      - name: Verify that examples build
```

Comment:
Don't you think it could deserve it's own job in the workflow with a dedicated step for each these commands ?

Reply:
I don't know. This would require a "workflow_run" trigger to operate after the previous steps, but yes, essentually why not? If you feel strongly about that, please propose a follow-up PR.